api/v1alpha/istio-manifests/1.4/authz/allow-data-producers-policy.yaml

```diff
@@ -7,6 +7,31 @@ metadata:
 spec:
   action: ALLOW
   rules:
+  - to:
+    - operation:
+      methods: ["GET,POST,PUT,DELETE"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/accounts*, /v1alpha/projects/$PROJECT_ID/accounts/*"]
+    - operation:
+      methods: ["GET,POST,PUT,DELETE"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/policies*, /v1alpha/projects/$PROJECT_ID/policies/*"]
+    - operation:
+      methods: ["GET,POST,PUT,DELETE"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/datasets*, /v1alpha/projects/$PROJECT_ID/datasets/*"]
+    - operation:
+      methods: ["POST"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/admin*, /v1alpha/projects/$PROJECT_ID/admin/*"]
+    - operation:
+      methods: ["POST"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/auth*, /v1alpha/projects/$PROJECT_ID/auth/*"]
```

Comment:
I would make this the full URL `/v1alpha/projects/$PROJECT_ID/auth:isDataProducer` otherwise a *Data Producer* has access to all `auth::custom` methods.